app/src/renderer/vuex/modules/governance/proposals.js

```diff
@@ -94,6 +80,7 @@ export default ({ node }) => {
         Sentry.captureException(error)
         state.error = error
       }
+      return undefined
```

Comment:
why do we want to return `undefined`?

Reply:
we did as well before, it is just more explicit this way

Reply:
i think a number of our modules do not have any return statements ... ?

Reply:
yeah it's what was already happening under the hood, I was thinking is more direct for readers in this way